python/mp/effects/casino.py

```diff
@@ -14,8 +14,8 @@ class Casino(effect.Effect):
     # Wish there were a better way than requiring this every time
     dm = copy.deepcopy(effect.Effect.dm)
 
-    def __init__(self, bead_set, color=color.Color(), duration=None, direction=1):
-        super().__init__("casino", bead_set, color=color, duration=duration)
+    def __init__(self, bead_set, color=color.Color(), duration=None, direction=1, **kwargs):
+        super().__init__(name="casino", bead_set=bead_set, color=color, duration=duration, **kwargs)
```

Comment:
Correct. Unless I missed any, I should have done that for whatever effects we already have.